### DISCUSSION TITLE
Why is it being redirected with trailingSlash?

### DISCUSSION THREAD

**Discussion by johnyejin** [nuxt/nuxt#18989] [Questions (Nuxt 2)] [answered]
Hi there 👋

I'm currently building a website with Nuxt2, but I'm facing an issue when routing with nuxt-child.

The folder structure is as follows.
```
├── pages
│   ├── index.vue
│   ├── order.vue
│   └── order
│       ├── global.vue
│       └── index.vue
```

There is no problem with CSR rendering, but when rendering with SSR, trailingSlash is attached like this.

1. `{domain}/order` with SSR rendering
2. redirected to `{domain}/order/`

I don't want a trailingSlash to be stuck after path.
Why is this happening and is there an appropriate way to solve this problem?

**Comment by TheAlexLichter** [comment, marked as answer]
IIRC this is how Vue Router back then did it. This is "resolved" in Vue 3 / Vue Router v4

You can also use the `trailingSlash` option.
